docs/docusaurus.config.ts

```diff
@@ -121,6 +121,10 @@ const config: Config = {
                   href: 'https://legacy-docs.dagster.io',
                   label: '1.9.9 and earlier',
                 },
+                {
+                  href: 'https://main.archive.dagster-docs.io/',
+                  label: 'Upcoming release (Preview)',
```

Comment:
@neverett thoughts on this being at the bottom of the drop down with this naming convention?

Reply:
I'm in favor of putting it at the bottom. Eventually, we'll have to deal with a very long dropdown list, but that's a problem for another day. I wish we could use "Preview", but I think we gotta drop it, since it's an overloaded term.